src/EventStore.Core/Helpers/LengthPrefixSuffixFramer.cs

```diff
@@ -96,7 +97,7 @@ public class LengthPrefixSuffixFramer {
 						_memStream.SetLength(_packageLength - PrefixLength); // remove suffix length
 						_memStream.Position = 0;
 
-						_packageHandler(_binaryReader);
+						_packageHandler?.Invoke(_binaryReader);
```

Comment:
if `_packageHandler` had a default action that was a noop then we could avoid the null check here (and in the two new framers)

Reply:
fixed :+1: